src/BaGet.Core/Configuration/StorageOptions.cs

```diff
@@ -12,5 +12,6 @@ public enum StorageType
         AwsS3 = 2,
         GoogleCloud = 3,
         Null = 4,
+        AliyunOSS = 5,
```

Comment:
For consistency let's lowercase the `ss` here:

```suggestion
        AliyunOss = 5,
```